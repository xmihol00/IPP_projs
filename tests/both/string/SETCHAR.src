
.IPPcode21

DEFVAR GF@rplc
MOVE GF@rplc string@&!

DEFVAR GF@str
MOVE GF@str string@AAAAAAAAAA

SETCHAR GF@str int@3 GF@rplc
WRITE GF@str
WRITE string@\010

SETCHAR GF@str int@9 string@BBB
WRITE GF@str
WRITE string@\010

SETCHAR GF@str int@0 GF@rplc
WRITE GF@str
WRITE string@\010
