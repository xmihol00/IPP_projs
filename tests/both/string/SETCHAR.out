AAA&AAAAAA
AAA&AAAAAB
&AA&AAAAAB
